spec/factories/book.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :book do
+    user
+    title { Faker::Book.title }
+    year_of_publication { Faker::Number.within(range: 500..2022) }
+    wikipedia_url { Faker::Internet.url }
+    goodreads_url { Faker::Internet.url }
+    author_id { Faker::Number.between(from: 1, to: Author.all.count) }
```

Comment:
why not just `author` (like `user` in the beginning)? This specifies association with another factory